datalad/distribution/tests/test_install.py

```diff
@@ -60,7 +65,13 @@ def test_installationpath_from_url():
               'http://example.com/lastbit',
               'http://example.com/lastbit.git',
               ):
-        assert_equal(_installationpath_from_url(p), 'lastbit')
+        assert_equal(_get_installationpath_from_url(p), 'lastbit')
+
+
+def test_get_git_url_from_source():
+    eq_(_get_git_url_from_source('///subds'), DATASETS_TOPURL + 'subds')
+    eq_(_get_git_url_from_source('http://example.com'), 'http://example.com')
+    assert_raises(NotImplementedError, _get_git_url_from_source, '//custom/subds')
 
```

Comment:
I think so. This one should be in effect only for our RIs iirc... try? :-)

On June 26, 2016 2:30:30 PM GMT+02:00, Benjamin Poldrack <notifications@github.com> wrote:
>> @@ -60,7 +65,13 @@ def test_installationpath_from_url():
>>                'http://example.com/lastbit',
>>                'http://example.com/lastbit.git',
>>                ):
>> -        assert_equal(_installationpath_from_url(p), 'lastbit')
>> +        assert_equal(_get_installationpath_from_url(p), 'lastbit')
>> +
>> +
>> +def test_get_git_url_from_source():
>> +    eq_(_get_git_url_from_source('///subds'), DATASETS_TOPURL +
>'subds')
>> +    eq_(_get_git_url_from_source('http://example.com'),
>'http://example.com')
>> +    assert_raises(NotImplementedError, _get_git_url_from_source,
>'//custom/subds')
>>  
>
>Still needs to work for local paths and SSHRI, too. Right?
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/datalad/datalad/pull/582/files/aeb215cac559b416f493727dfbd9ee8d8b0b5f03#r68502326